### ISSUE TITLE
Coundn't connect to Register Method

### ISSUE THREAD

**Issue by Bigbhardwaj** [cornflourblue/aspnet-core-registration-login-api#2]
Yes I figured it out soon..I Checked register method with below code and Postaman returns with success.
public IActionResult Register([FromForm]UserDto userDto)
{
// map dto to entity
var user = _mapper.Map(userDto);
return Ok(new
{
user
});
but if exact code is in place in Register method. then the execution flow hit by another blocker.
Postman request

![image](https://user-images.githubusercontent.com/9026778/47103169-43bbb780-d25c-11e8-8183-91402e83a1da.png)

Exception : 

![image](https://user-images.githubusercontent.com/9026778/47103187-53d39700-d25c-11e8-8843-12f9289ac775.png)

**Comment by Rackar** [comment]
> Yes I figured it out soon..I Checked register method with below code and Postaman returns with success.
> public IActionResult Register([FromForm]UserDto userDto)
> {
> // map dto to entity
> var user = _mapper.Map(userDto);
> return Ok(new
> {
> user
> });
> but if exact code is in place in Register method. then the execution flow hit by another blocker.
> Postman request
> 
> ![image](https://user-images.githubusercontent.com/9026778/47103169-43bbb780-d25c-11e8-8183-91402e83a1da.png)
> 
> Exception :
> 
> ![image](https://user-images.githubusercontent.com/9026778/47103187-53d39700-d25c-11e8-8843-12f9289ac775.png)

No,the Postman returned a '404 not found', check the api path in program.cs, 

```public static IWebHost BuildWebHost(string[] args) =>
            WebHost.CreateDefaultBuilder(args)
                .UseStartup<Startup>()
                .UseUrls("http://localhost:4000")
                .Build();
    }```
thats "http://localhost:4000/users/register"
